File: 프로그래머스/0/181943. 문자열 겹쳐쓰기/문자열 겹쳐쓰기.java
class Solution {
    public String solution(String my_string, String overwrite_string, int s) {
        String answer = "";
        int cnt = 0;
        
        for(int i = 0; i < my_string.length(); i++) {
            if(s <= i && i < s + overwrite_string.length()) {
                answer += overwrite_string.charAt(cnt++);
            }
            else answer += my_string.charAt(i);
        }
        
        return answer;
    }
}

File: 프로그래머스/0/181943. 문자열 겹쳐쓰기/문자열 겹쳐쓰기.cs
using System.Text.RegularExpressions;

public class Solution {
    public string solution(string my_string, string overwrite_string, int s) {
        my_string = my_string.Remove(s, overwrite_string.Length);
        string answer = my_string.Insert(s, overwrite_string);
        return answer;
    }
}